docs/pages/project/contributing/contributing-ui.md

```diff
@@ -96,6 +96,27 @@ make build-ui
 
 > Changes are not recompiled directly, you will have to run to rebuild the UI to see them
 
+
+#### Build Meshery-UI
+
+```
+make build-meshery-ui
+```
+
+> While `make build-ui` builds all user interfaces on your local machine. 
+
+> `make build-meshery-ui` builds only meshery user interface on your local machine.
+
+
+#### Run Meshery
+To start running Meshery locally:
+```
+make run-fast
```

Comment:
@utkarsh-pro should we displace the contents of `make run-local` with make run-fast and consolidate into one command?

Reply:
Yes, I think we should do that.

Reply:
Sounds good, @utkarsh-pro 

@warunicorn19 will you do this:

<img width="222" alt="Screen Shot 2021-07-27 at 8 15 28 AM" src="https://user-images.githubusercontent.com/7570704/127159820-1747cb90-984e-4711-9119-72dbb871553f.png">

Reply:
Yup I'll do it right away

Reply:
@leecalcote just to be clear, i have to make an issue and reference this over there right?

Reply:
Right. 
![FD168B5B-08D0-41E1-82D2-ECC57E1FBEDC](https://user-images.githubusercontent.com/7570704/127472819-f7e37e0f-3e2d-4646-8bcc-ea000557e1ee.jpeg)

Reply:
@leecalcote already did that #3656

Reply:
Oh! 🙂